config/application.rb

```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
     config.active_job.queue_adapter = :sidekiq
+
+    config.autoload_paths += %W(#{config.root}/lib)
+    config.autoload_paths += %W(#{config.root}/app/models/question)
```

Comment:
It seems that this is needed because of `FactoryGirl`. What do you think about being explicit in the Factory with:

```ruby
require_relative '../../app/models/question/boolean_question'
require_relative '../../app/models/question/free_text_question'
```
And we don't autoload at the application level?